packages/Checkboxes/src/index.tsx

```diff
@@ -46,14 +49,17 @@ const Checkboxes: FunctionComponent<ICheckboxesProps> = ({
             show = true,
             id = uuidv1()
           } = item;
+
           if (!show) {
-            return <Fragment key={id} />;
+            return <Fragment key={uuidv1()} />;
           }
+
           return (
             <div
+              data-testid={`checkbox-${key + 1}`}
               key={id}
               className={`form-field form-field--checkbox ${
```

Comment:
Could we use here classnames package instead to enhace more readable dynamic classnames?

Reply:
fixed